functional/FunctionalTest.java

```diff
@@ -2421,16 +2421,9 @@ public static void testSetBucketPolicy(String objectPrefix, PolicyType policyTyp
         client.putObject(bucketName, objectName, new ByteArrayInputStream(data), data.length, customContentType);
       }
 
+      // Authenticated download is not supported in AWS S3.
```

Comment:
Unable to understand this comment.

Reply:
Where do you see it is not supported?  You could give a doc reference

Reply:
It is not documented @balamurugana